tests/tests_tensorflow/datasets/test_tf_ooddataset.py

```diff
@@ -98,23 +74,17 @@ def test_instanciate_ood_dataset(backend, as_supervised, expected_output):
     [
         ("tensorflow", True, [200, 2, 3, 0.5]),
```

Comment:
If only "tensorflow" backend is used in this file, can we remove it from the pytest parametrization?